packages/web-app-markdown-editor/src/App.vue

```diff
@@ -98,14 +143,44 @@ export default {
       return loadFileTask.isRunning || saveFileTask.isRunning
     })
 
+    const config = computed(() => {
+      const { showPreview = true } = getCurrentInstance().proxy.applicationConfig
+      return { showPreview }
+    })
+
     onMounted(() => {
-      loadFileTask.perform(getCurrentInstance().proxy)
+      const that = getCurrentInstance().proxy
+      loadFileTask.perform(that)
```

Comment:
Again there's no need to use the instance imho, as all we need is `getFileContents` which is available from the instance via `useAppDefaults`, so we can use that directly.

Just pass `defaults` (from my recommendation above) to perform.